packages/core/src/server.ts

```diff
@@ -29,15 +28,13 @@ export class Server extends Context {
       await sequence.run(this, req, res);
     });
 
-    // NOTE(bajtos) bluebird.promisify looses type information about the original function
-    // As a (temporary?) workaround, I am casting the result to "any function"
-    // This would be a more accurate type: (port: number) => Promise<http.Server>
-    const listen = bluebird.promisify(server.listen, {context: server}) as Function;
-    await listen(this.config.port);
+    server.listen(this.config.port);
```

Comment:
Interestingly enough, the ephemeral port `0` is resolved into a real port value even before the server is listening. SuperTest is using this fact to provide sync API for creating the client (Test) instance.